scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models currently used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
```

Comment:
I'm noticing that the word `currently` is used in a few places, but it seems its use is a bit superfluously? I'm not sure it's adding anything to the sentence/understanding. So I think they can be removed in most of the spots it's used. Your discretion, of course! Just a suggestion.

Reply:
Removed the unnecessary `currently`s